libs/components/forms/src/lib/modules/file-attachment/file-attachment.component.html

```diff
@@ -2,9 +2,15 @@
   <div
     class="sky-file-attachment-label-wrapper"
     [attr.id]="labelElementId"
-    [ngClass]="{ 'sky-control-label-required': required && hasLabelComponent }"
+    [ngClass]="{
+      'sky-control-label-required': required && (hasLabelComponent || labelText)
+    }"
   >
-    <ng-container *ngTemplateOutlet="labelContent"></ng-container>
+    <ng-container *ngIf="labelText; else labelContent">
+      <ng-container *ngIf="!labelHidden">
```

Comment:
The `*ngIf="!labelHidden"` can go on the `span`; there's no need for the wrapping `ng-container`.